README.md

```diff
@@ -75,12 +75,15 @@ Role Variables
 
     - `volumes`: a list of volumes to attach to the VM.  Each volume is
       defined with the following dict:
+        - `type`: Type of the volume. Options include `volume`, `file` and `network`.
         - `pool`: Name or UUID of the storage pool from which the volume should be
-          allocated.
+          allocated. Required when `type` is `volume`.
         - `name`: Name to associate with the volume being created; For `file` type volumes include extension if you would like volumes created with one.
         - `file_path`: Where the image of `file` type volumes should be placed; defaults to `libvirt_volume_default_images_path`
         - `device`: `disk` or `cdrom`
-        - `capacity`: volume capacity (can be suffixed with M,G,T or MB,GB,TB, etc) (required when type is `disk`)
+        - `capacity`: volume capacity, can be suffixed with k, M, G, T, P or E when type is `network` or MB,GB,TB, etc when type is `file` (required when type is `file`)
```

Comment:
Looks like this is also required when type is network.

Reply:
Updated in latest commit